xml/System.IO.Pipelines/ReadResult.xml

```diff
@@ -112,9 +111,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>
-          <see langword="true" /> if the <see cref="T:System.IO.Pipelines.PipeReader" /> is complete; otherwise, <see langword="false" />.</summary>
```

Comment:
This line was originally automatically ported from the triple slash comments, but I don't think it's right to say the `PipeReader` is what was completed. I think we should be mentioning the `ReadAsync` call was the one that completed. Is this a correct assumption, @davidfowl ?

Reply:
Actually it *is* the `PipeReader` that is completed, though that terminology may need some refinement. The `IsCompleted` property will be true when the **writer** on the other side calls `.Complete` to indicate that no further data will be written. So this is a signal to the caller of `ReadAsync` to tell them to stop reading, as no more data will be written.